plugins/wfp/common.py

```diff
@@ -512,3 +516,18 @@ def missed_followup_visit(self, visits, formIds, next_visit__date__, secondNextV
                 ):
                     count_missed_visit = count_missed_visit + 1
         return count_missed_visit
+
+    def calculate_birth_date(self, current_record):
+        age_entry = current_record.get("age_entry", None)
+        age = current_record.get("age__int__", None)
+        registration_date = current_record.get("registration_date", None)
+        calculated_date = None
+        if age_entry is not None and age_entry != "":
+            beneficiary_age = int(age)
+            registered_at = datetime.strptime(registration_date[:10], "%Y-%m-%d").date()
+            if age_entry == "years":
+                calculated_date = registered_at - timedelta(days=(beneficiary_age * 365.25) + 1)
```

Comment:
Not sure why we don't use it yet, but if we'd use `python-dateutil` this would be easier, like
```python
 calculated_date = registered_at - relativedelta(years=beneficiary_age)
```
What do you think, worth it? https://github.com/dateutil/dateutil

Reply:
Thanks, I will change and use `dateutil` which is more simple.